internal/fs/fs.go

```diff
@@ -376,3 +376,26 @@ func IsSymlink(path string) (bool, error) {
 
 	return l.Mode()&os.ModeSymlink == os.ModeSymlink, nil
 }
+
+// ResolvePath resolves the path of the given symlink. If the given path is not a symlink,
+// then it returns the same path.
+func ResolvePath(path string) (string, error) {
```

Comment:
It's totally useless. 😅
I just noticed I already call `IsSymlink()` before `ResolvePath()`.

I'll remove it when I've time to refactor the PR.